x-pack/plugins/siem/public/components/page/hosts/hosts_table/columns.tsx

```diff
@@ -72,7 +73,14 @@ export const getHostsColumns = (
   },
   {
     field: 'node.lastSeen',
-    name: i18n.LAST_SEEN,
+    name: (
+      <EuiToolTip content={i18n.FIRST_LAST_SEEN_TOOLTIP}>
+        <span>
+          {i18n.LAST_SEEN}
+          <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+        </span>
+      </EuiToolTip>
+    ),
```

Comment:
For this area and `x-pack/plugins/siem/public/components/page/hosts/hosts_table/columns.tsx`

I tried this out and it looks like we can get away without using the `span` which would reduce some DOM weight for us.

```ts
 name: (
      <EuiToolTip content={i18n.FIRST_LAST_SEEN_TOOLTIP}>
        <>
          {i18n.LAST_SEEN}
          <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
        </>
      </EuiToolTip>
    ),
```